provider/google.go

```diff
@@ -238,14 +259,10 @@ func (p *GoogleProvider) submitChange(change *dns.Change) error {
 	}
 
 	for _, del := range change.Deletions {
-		log.Infof("Del records: %s %s %s", del.Name, del.Type, del.Rrdatas)
+		log.Debugf("Plan requests record deletion: %s %s %s %d", del.Name, del.Type, del.Rrdatas, del.Ttl)
 	}
 	for _, add := range change.Additions {
-		log.Infof("Add records: %s %s %s", add.Name, add.Type, add.Rrdatas)
-	}
-
-	if p.dryRun {
-		return nil
+		log.Debugf("Plan requests record addition: %s %s %s %d", add.Name, add.Type, add.Rrdatas, add.Ttl)
```

Comment:
why this debug statement is needed here, I think it would be either logged in: 

1. separateChange() - if no matching zone is found
2. below in this method with at line ~277-284 with the zone where it would be created or deleted from